actions/setup/action.yml

```diff
@@ -17,6 +12,13 @@ inputs:
     required: false
     default: '1.22'
 
+  cacheKey:
```

Comment:
Is this a cache prefix rather than a cache key?

Reply:
Yes, but we already had this input since before so I didn't want to rename it.

Reply:
You can see here where `cacheKey` is used: https://github.com/search?q=org%3Aeinride+cacheKey+language%3AYAML+path%3A%2F%5E%5C.github%5C%2Fworkflows%5C%2F%2F&type=code